client/src/ui/components/Button.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import cx from 'classnames';
+
+interface ButtonProps {
```

Comment:
the best would be to extend the button props and forward them to the button using `{...props}` so we don't have to redefine onClick etc..

Reply:
@pradel  resolved both feedbacks in this one.

Reply:
@Akirtovskis on this one I meant `interface ButtonProps extends React.HTMLProps<HTMLButtonElement>`, so that way we don't have to define onClick at all :)

Reply:
@pradel is this resolved or still in question? It didn't work out of the box like you suggested, so I assume that's why it is resolved.